docs/external-node.md

```diff
@@ -0,0 +1,532 @@
+host to the external Node. These rules can be set with the item 
+`policyBypassRules` in the agent  configuration. Following is a configuration
+example,
+
+```yaml
+policyBypassRules:
+  - direction: ingress
+    protocol: tcp
+    cidr: 1.1.1.1/32
+    port: 22
+```
+
+The `direction` in one policyBypassRule supports `ingress` or `egress`. And
+the supported protocols include, `tcp`,`udp`, `icmp` and `ip`. The `cidr`
+gives the peer address, it is the destination in an `egress` rule, and the
+source in an `ingress` rule. For `tcp` and `udp` protocols, the `port` is
+required to describe the destination port in the packet.
+
+The policyBypassRules are processed before Antrea NetworkPolicy rules when
+forwarding the packets in OVS datapath, and the disposition of the rules is to
+allow the packets.
+
+# RBAC for antrea-agent
+
+An external Node is thought as an untrusted point on the network. To follow the
+security principle that least permissions are given, the RBAC for `antrea-agent`
+running on `ExternalNode` feature is modified:
+
+- Only `get`, `list` and `watch` permissions are given on resource `ExternalNode`
+- Only `update` permission is given on resource `antreaagentinfos`, and `create`
+  permission is moved to `antrea-controller`
+- for more details please refer to [vm-agent.yaml](../build/yamls/externalnode/vm-agent.yaml)
+
+`antrea-agent` reports its status by updating the `antreaagentinfo` resource
+which is configured with the same name as the `ExternalNode`. `antrea-controller`
+creates an `antreaagentinfo` object for each new `ExternalNode`, and then
+`antrea-agent` updates it every minute with its latest status. `antreaagentinfo`
+is deleted by `antrea-controller` when the `ExternalNode` is removed.
+
+# VM network configuration
+
+`antrea-agent` uses the interface IPs or name to find the network interface on
+the external Node, and then attaches it on the OVS bridge. The network interface
+is attached on OVS as uplink, and a new OVS internal Port is added to take over
+the uplink interface's IP/MAC and routing configurations. On Windows, the DNS
+configurations are also moved to the OVS internal port from uplink. Before
+attaching the uplink to OVS, the network interface is renamed with a suffix
+"~", and OVS internal port is configured with the original name of the uplink.
+As a result, IP/MAC/routing entries are seen on a network interface configuring
+with the same name on the external Node.
+
+The outbound traffic sent from the external Node enters OVS from the internal
+port, and finally output from the uplink, and the inbound traffic enters OVS
+from the uplink and output to the internal port. The IP packet is processed by
+the OpenFlow pipeline, and the non-IP packet is forwarded directly.
+
+Following is a diagram for the traffic model on an external Node.
+![Traffic On ExternalNode](assets/traffic_external_node.svg)
+
+# OpenFlow pipeline
```

Comment:
maybe make this section the last one in the document

Reply:
Do you mean OpenFlow pipeline is put after `Limitation`?